tutorials/MD/data/md.mdp

```diff
@@ -0,0 +1,314 @@
+vdw-modifier             = Potential-shift-Verlet
+; cut-off lengths       
+rvdw-switch              = 0
+rvdw                     = 1
+; Apply long range dispersion corrections for Energy and Pressure
+DispCorr                 = No
+; Extension of the potential lookup tables beyond the cut-off
+table-extension          = 1
+; Separate tables between energy group pairs
+energygrp-table          = 
+; Spacing for the PME/PPPM FFT grid
+fourierspacing           = 0.12
+; FFT grid size, when a value is 0 fourierspacing will be used
+fourier_nx               = 0
+fourier_ny               = 0
+fourier_nz               = 0
+; EWALD/PME/PPPM parameters
+pme_order                = 4
+ewald_rtol               = 1e-05
+ewald-rtol-lj            = 0.001
+lj-pme-comb-rule         = Geometric
+ewald_geometry           = 3d
+epsilon_surface          = 0
+implicit_solvent         = No
+
+; OPTIONS FOR WEAK COUPLING ALGORITHMS
+; Temperature coupling  
+tcoupl                   = Nose-Hoover
+nsttcouple               = -1
+nh-chain-length          = 10
+print-nose-hoover-chain-variables = no
+; Groups to couple separately
+tc-grps                  = Protein Non-Protein
+; Time constant (ps) and reference temperature (K)
+tau-t                    = 0.5 0.5
+ref-t                    = 300. 300.0
+; pressure coupling     
+Pcoupl                   = Parrinello-Rahman
+Pcoupltype               = Isotropic
+nstpcouple               = -1
+; Time constant (ps), compressibility (1/bar) and reference P (bar)
+tau-p                    = 1.0
+compressibility          = 4.6E-5
+ref-p                    = 1.0
+; Scaling of reference coordinates, No, All or COM
+refcoord_scaling         = No
+
+; OPTIONS FOR QMMM calculations
+QMMM                     = no
+; Groups treated Quantum Mechanically
+QMMM-grps                = 
+; QM method             
+QMmethod                 = 
+; QMMM scheme           
+QMMMscheme               = normal
+; QM basisset           
+QMbasis                  = 
+; QM charge             
+QMcharge                 = 
+; QM multiplicity       
+QMmult                   = 
+; Surface Hopping       
+SH                       = 
+; CAS space options     
+CASorbitals              = 
+CASelectrons             = 
+SAon                     = 
+SAoff                    = 
+SAsteps                  = 
+; Scale factor for MM charges
+MMChargeScaleFactor      = 1
+
+; SIMULATED ANNEALING  
+; Type of annealing for each temperature group (no/single/periodic)
+annealing                = 
+; Number of time points to use for specifying annealing in each group
+annealing-npoints        = 
+; List of times at the annealing points for each group
+annealing-time           = 
+; Temp. at each annealing point, for each group.
+annealing-temp           = 
+
+; GENERATE VELOCITIES FOR STARTUP RUN
+gen-vel                  = yes
+gen-temp                 = 300
+gen-seed                 = 173529
+
+; OPTIONS FOR BONDS    
+constraints              = all-bonds
```

Comment:
So maybe worth a longer discussion here, but it would definitely be worth highlighting in teaching these mdps the issues with the historical use of `all-bonds` constraints and why they are not recommended for most modern ffs (in this case I think gromos might have actually been paramterised for it, someone else can check here). In most cases `all-bond` constraints only serve to add an extra layer of atrificiality to the system without any performance gains (it is my understanding that `all-bond` constraints actually run slower than `h-bonds`). That being said there are cases where `all-bonds` are necessary, primarily when using VS or HMR methods to get dt to ~ 4 fs.

I'll refer back to this change in grompp behaviour as of gromacs 2019: http://manual.gromacs.org/documentation/2019/release-notes/2019/major/miscellaneous.html#grompp-discourages-use-of-constraints-all-bonds

Probably too late to address for monday, but tagging @philbiggin here for future iterations of the course.